cloud/test/meta_service_job_test.cpp

```diff
@@ -58,6 +58,7 @@ void start_compaction_job(MetaService* meta_service, int64_t tablet_id, const st
     if (input_version.first > 0 && input_version.second > 0) {
         compaction->add_input_versions(input_version.first);
         compaction->add_input_versions(input_version.second);
+        compaction->set_judge_input_versions_range(true);
```

Comment:
what if the old version of BEs?

Reply:
The parameter is used to replace confi `enable_parallel_cumu_compaction`.
In old version of BEs, only if enable_parallel_cumu_compaction is true, input_versions will be set.
In new version of Bes, input_versions always are set, and use other param to replace `enable_parallel_cumu_compaction`.
Compatibility is also handled here.